<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DefaultUP</class>
 <widget class="QWidget" name="DefaultUP">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>550</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Student Number</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>140</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Username</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>140</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Password</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="IDlineEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>180</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="UserlineEdit">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>180</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="PasslineEdit">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>180</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="CopyToFileButton">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>250</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Next</string>
   </property>
  </widget>
  <widget class="QPushButton" name="DefaultsFinished">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>300</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Done</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Entering information for user number:</string>
   </property>
  </widget>
  <widget class="QLabel" name="numberLable">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>40</y>
     <width>21</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>140</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Mojudi</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="MojudilineEdit">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>180</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
